jdaviz/core/tools.py

```diff
@@ -1,6 +1,7 @@
 import os
 import time
 
+import astropy.units as u
 import numpy as np
```

Comment:
Small nitpick to make import style consistent throughout the package.

```suggestion
import numpy as np
from astropy import units as u
```